pkg/controllers/subnet/subnet_controller.go

```diff
@@ -76,7 +77,10 @@ func (r *SubnetReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 				obj.Spec.IPv4SubnetSize = vpcNetworkConfig.DefaultIPv4SubnetSize
 			}
 		}
-		if _, err := r.Service.CreateOrUpdateSubnet(obj, nil); err != nil {
+		var tags []model.Tag
+		tags = append(tags, model.Tag{Scope: servicecommon.String(servicecommon.TagSubnetFor),
```

Comment:
we add sx-op/subnet-for:LabelDefaultVMSubnet here for the subnet, how we know this subnet is created for VM instead of pod?

Reply:
ditto